themes/shijin4/layouts/blog/single.html

```diff
@@ -1,8 +1,8 @@
 {{ partial "header.html" . }}
 <div id="page">
 {{ partial "navbar.html" . }}
-	<div class="row" id="main-front">
-		<div class="col-md-9 col-md-push-3">
+	<div class="row" id="main-front">		
```

Comment:
```suggestion
	<div class="row" id="main-front">
```
Trailing whitespace here.